core/frontend/src/render/webgl/Texture.ts

```diff
@@ -277,15 +275,8 @@ class Texture2DCreateParams {
       }
     }
 
-    // Always draw to canvas for ImageBitmap
-    const canvas = document.createElement("canvas");
-    canvas.width = targetWidth;
-    canvas.height = targetHeight;
-    const context = canvas.getContext("2d")!;
-    context.drawImage(image, 0, 0, canvas.width, canvas.height);
-
```

Comment:
Removing the use of canvas reduces significantly time spent in TexImage2D on Firefox.

However, I am unsure whether it might not work as expected if ``targetWidth``/``targetHeight`` is different from the ``ImageBitmap`` original dimensions. Ideally, this should have been handled at the bitmap creation (``createImageBitmap`` should allow to specify new dimension in its options), but this case could still happen.